dists/fs-repo-migrations/matrices/v1.0.0

```diff
@@ -0,0 +1,10 @@
+darwin 386
```

Comment:
Why do we need these? As far as I know 32 bit is not used on any current OS X system that is able to run these binaries (and pretty much nobody, not even [go-lang](https://golang.org/dl/) or [node.js](https://nodejs.org/en/download/) give you 32 bit binaries for OS X)